### DISCUSSION TITLE
Paper Minecraft v11.7 (Minecraft 2D)

### DISCUSSION THREAD

**Discussion by MichaelXDlol** [leopard-js/leopard-mentors#17] [Cleanup Crew]
### Scratch Project URL

https://scratch.mit.edu/projects/10128407

### Leopard Project URL

https://leopardjs.com/projects/rDsxNWwawK

### What kind of error(s) are you experiencing?

- [ ] Leopard is failing to generate JavaScript code
- [ ] The JavaScript code generated by Leopard doesn't work correctly
- [ ] The JavaScript code generated by Leopard is confusing or needs improvement
- [ ] Other

### Description of issue(s)

White screen

**Comment by 5pagg3tt1** [comment]
very interesting
